readme.md

```diff
@@ -827,6 +827,7 @@
 - [Sketch](https://github.com/diessica/awesome-sketch#readme) - Design app for macOS.
 - [Boilerplate Projects](https://github.com/melvin0008/awesome-projects-boilerplates#readme)
 - [Readme](https://github.com/matiassingers/awesome-readme#readme)
+- [Wiki](https://github.com/MyHoneyBadger/awesome-github-wiki#readme) - Additional, more detailed than README, documentation on GitHub.
```

Comment:
Incorrect title

Reply:
And I think the description could be improved.

Reply:
Thanks, @sindresorhus 
Title fixed, Could you please elaborate what can be improved.

Reply:
I would rewrite the sentence to not use commas and change around the order. Currently it doesn't read that well.

Reply:
Something like:

> Comprehensive GitHub documentation beyond README.